plutus-pab/src/Plutus/PAB/Webserver/Types.hs

```diff
@@ -100,7 +99,6 @@ data InstanceStatusToClient
 data CombinedWSStreamToClient
     = InstanceUpdate ContractInstanceId InstanceStatusToClient
     | SlotChange Slot -- ^ New slot number
-    | WalletFundsChange Wallet Value -- ^ The funds of the wallet have changed
```

Comment:
Removing this constructor impacts the marlowe-dashboard-client module `MainFrame.State` in a significant way I think. How should I update Marlowe?

Reply:
Generally I think removing the constructor is fine - information about the wallet funds should really come from the wallet, not the PAB. And Daedalus for example just polls the wallet backend's endpoint for wallet funds regularly (it's quite cheap). What do you think @nau ?